src/objects/zcl_abapgit_object_iobj.clas.abap

```diff
@@ -89,7 +107,25 @@ CLASS zcl_abapgit_object_iobj IMPLEMENTATION.
     DATA: ls_details TYPE bapi6108,
           lt_infoobj TYPE STANDARD TABLE OF bapi6108io,
           ls_return  TYPE bapiret2,
-          lt_return  TYPE STANDARD TABLE OF bapiret2.
+          lt_return  TYPE STANDARD TABLE OF bapiret2,
+          lt_comptab TYPE STANDARD TABLE OF spam_cvers.
+
+    CALL FUNCTION 'OCS_GET_INSTALLED_COMPS'
```

Comment:
suggest not adding this part, just try executing the code, if it fails then throw an error

Reply:
Done